server/bin/test-bin/selinux.py

```diff
@@ -0,0 +1,17 @@
+# This is a mock file for selinux.restorecon test cases
```

Comment:
Great idea for this mock module.  Can we create a new directory, `test-lib`, as a sister directory to `test-bin`, moving this module there, and adding it to the `PYTHONPATH` from there in `unittests`?  I don't think it needs to be installed in the test environment, but it probably should have an empty `__init__.py` to be a proper python3 module.

Reply:
Ok sure will do that